run-it/single-tenant-host/README.md

```diff
@@ -1,8 +1,10 @@
 # Docker Host Single Tenant (ST)
 
-To run Parabol in single tenant mode (e.g. simple docker-compose on a docker host).
+> ⚠️ **Parabol does not provide any support on this stack**. Use it under your own resposibility.
```

Comment:
Our license already covers this, I'd opt for fewer words here & remove because no one likes reading disclaimers.

Reply:
Ok!

Reply:
Security scripts are still in use on the `parabol.dockerfile`, which will be removed, along these scripts, as soon as we upgrade all our PPMIs. I hope we can do that sooner than later.